RetailCoder.VBE/UI/Command/CodeExplorerCommand.cs

```diff
@@ -18,10 +18,12 @@ public CodeExplorerCommand(IDockablePresenter presenter)
             _presenter = presenter;
         }
 
-        public override RubberduckHotkey Hotkey
+        public override HotkeySetting DefaultHotkey => new HotkeySetting(typeof(CodeExplorerCommand))
```

Comment:
interesting idea, but IMO this need not be a concern for a Command. This is a separate thing that should stay maintained in the `HotkeySettings` instead. Or we should start bundling RD with a default configuration and enable the user to "Reset Settings". This makes default-changes a configuration change instead of a possibly code-breaking change....

Reply:
@retailcoder opinions on ^^?

Reply:
Agreed, settings shouldn't be on the command interface, similar to inspections: settings determine what state the class holds, but remain separate from the class they're configuring.